File: 1051-height-checker/1051-height-checker.js
/**
 * @param {number[]} heights
 * @return {number}
 */
var heightChecker = function(heights) {
  const sorted = heights.slice()
  sorted.sort((a, b) => a - b)
  let ans = 0
  for (let i = 0; i < heights.length; i++) {
    ans += heights[i] != sorted[i]
  }
  return ans
};

File: 1051-height-checker/1051-height-checker.go
func heightChecker(heights []int) int {
  expected := make([]int, len(heights))
  copy(expected, heights)
  sort.Ints(expected)
  ans := 0
  for i := 0; i < len(heights); i++ {
    if expected[i] != heights[i] {
      ans ++
    }
  }
  return ans
}